packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.html

```diff
@@ -23,25 +23,16 @@
                     data-label="{{:: 'ip_table_header_service' | translate }}"
                     data-size="xl"
                 >
-                    <label class="oui-select">
-                        <select
-                            id="service"
-                            name="service"
-                            class="oui-select__input"
-                            data-ng-model="$ctrl.model.selectedService"
-                            data-ng-options="service as service.displayName group by $ctrl.getServiceTypeLabel(service.type) for service in $ctrl.services | orderBy:'displayName' track by service.serviceName"
-                        >
-                            <option
-                                data-ng-disabled="true"
-                                data-translate="global_select"
-                            >
-                            </option>
-                        </select>
-                        <span
-                            class="oui-icon oui-icon-chevron-down"
-                            aria-hidden="true"
-                        ></span>
-                    </label>
+                    <oui-select
+                        items="$ctrl.services"
+                        match="displayName"
+                        model="$ctrl.model.selectedService"
+                        name="service"
+                        placeholder="{{:: 'global_select' | translate}}"
+                        group-by="$ctrl.constructor.getServiceTypeLabel"
```

Comment:
```suggestion
                        group-by="$ctrl.constructor.getServiceTypeLabel(service.type)"
```

Reply:
that is not working with oui-select as the directive pass automatically the item as parameter for the function. We cannot set any other variable. This is why I had to change the way the translations are set

Reply:
okay